docker/autoware-openadk/Dockerfile

```
@@ -0,0 +1,137 @@
+# hadolint global ignore=DL3006,DL3008,DL3009,DL3015,DL3013,DL3027,DL3042
+ARG BASE_IMAGE
+
+FROM $BASE_IMAGE as base
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+ARG ROS_DISTRO
+
+# Install apt packages
+# hadolint ignore=DL3008
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends \
+  git \
+  ssh \
+  wget \
+  cmake \
+  curl \
+  gosu \
+  ccache \
```

```suggestion
  ccache \
```

I think we should move this back to ansible, the normal users need this on their machines with the `setup-dev-env.sh` as well.

---

You're right